### PR TITLE
v1.0.0

### PR DESCRIPTION
**Pull request by mxstbr**
- [x] Update CHANGELOG
- [ ] Deploy version of website with `moduleName`
- [x] Publish v1.0.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+### 1.0.0
+* added shared stylelint config to configure a couple of rules to fit with styled-components’ style of writing CSS
```
Comment (ismay) [inline CHANGELOG.md]: Wasn't this added in https://github.com/styled-components/stylelint-processor-styled-components/commit/cf988ad492586def7ca2454a161ad216aa280129, which was released with 0.4.0?

Comment (mxstbr) [reply CHANGELOG.md]: Good point, moved it!

**Comment by mxstbr** [comment]
Published! Now just gotta do all that other jazz...

**Comment by mxstbr** [comment]
It's oooouuuutttt!!

https://medium.com/styled-components/announcing-production-ready-linting-for-styled-components-58adcdc4a97
https://spectrum.chat/thread/0b38ed2c-82e0-468b-bbc2-4b2ba5752f39

**Comment by ismay** [comment]
Woohoo! Great! 😃 🎉 🎈🎉 🎉

**Pull request merged**